scenarios/fuel_test/src/AllEventsHandlers.ts

```diff
@@ -241,15 +241,15 @@ AllEvents.Transfer.handler(async ({ event }) => {
   );
 });
 
-const callSchema = S.object({
-  assetId: S.string,
-  to: S.string,
-  amount: SExtra.bigint,
-});
-AllEvents.Call.handler(
-  async ({ event }) => {
-    callSchema.assert(event.params)!;
-    expectType<AssertSchemaType<typeof event.params, typeof callSchema>>(true);
-  },
-  { wildcard: true }
-);
+// const callSchema = S.object({
```

Comment:
Is there a reason you are leaving this here?

Reply:
I wanted to keep it, so I can easily uncomment for future testing